test/common/http/http2/http2_frame.cc

```diff
@@ -10,8 +10,8 @@
 
 namespace {
 
-// Make request stream ID in the network byte order
-uint32_t makeRequestStreamId(uint32_t stream_id) { return htonl((stream_id << 1) | 1); }
+// Converts stream ID to the network byte order.
+uint32_t makeStreamId(uint32_t stream_id) { return htonl(stream_id); }
```

Comment:
makeNetworkByteStreamId()? makeNonClientStreamId()? I feel like method name should be more specific, so people reading other parts of the code knows what type of id is being made.

Reply:
Also, perhaps a quick comment about this (supports even values...?) etc.

Reply:
Done.